py/server/deephaven/table.py

```diff
@@ -1852,7 +1852,7 @@ def format_row_where(self, cond: str, formula: str) -> Table:
 
     def layout_hints(self, front: Union[str, List[str]] = None, back: Union[str, List[str]] = None,
                      freeze: Union[str, List[str]] = None, hide: Union[str, List[str]] = None,
-                     column_groups: List[dict] = None) -> Table:
+                     column_groups: List[dict] = None, search_display_mode: str = None) -> Table:
```

Comment:
@jmao-denver is there a better way to use an `enum` from in python? I tried using `jpy.get_type("io.deephaven.engine.util.LayoutHintBuilder.SearchDisplayModes")` but I get a Class not found error.

Reply:
it is an inner class, the JNI syntax is to use '$' notation.
```
io.deephaven.engine.util.LayoutHintBuilder$SearchDisplayModes
```
Please see this file for an example wrapping of an Enum.
```
py/server/deephaven/plot/linestyle.py
```

Reply:
I am actually not very sure about introducing an Enum in this case. I am fine with the parameter being a string with 3 valid, self-evident values.

Reply:
Okay, I've used an Enum, let me know if you want me to revert to string. I think @chipkent has previously said he prefers enums, not sure. Downside is you need a `from deephaven.table import SearchDisplayMode` before this.

Or I could make it accept both a string or the enum?